README.md

```diff
@@ -55,11 +55,13 @@ instance-1  us-central1-a  n1-standard-1               10.240.0.2   8.34.219.29
 
```

Comment:
can you create 2 sections under this title, 1 for debian and 1 for alpine.

For debian, INSTALL_COMPONENTS works, but for alpine it doesn't. Maybe just show how people can extend the alpine based image using a Dockerfile, with kubectl example?